README.md

```diff
@@ -29,14 +29,14 @@ from ndx_binned_spikes import BinnedAlignedSpikes
 data = np.array(
     [
         [  # Data of unit with index 0
-            [5, 1, 3, 2],  # Bin counts around the first timestamp
-            [6, 3, 4, 3],  # Bin counts around the second timestamp
-            [4, 2, 1, 4],  # Bin counts around the third timestamp
+            [5, 1, 3, 2],  # Bin counts around the first event's reference timestamp
```

Comment:
```suggestion
            [5, 1, 3, 2],  # Bin counts around the first event's timestamp
```
Do think this will be less or more clear?  I am confused about what is an event's reference.